owncloudApp/src/androidTest/java/com/owncloud/android/settings/more/OCSettingsMoreTest.kt

```diff
@@ -0,0 +1,177 @@
+ */
+
+package com.owncloud.android.settings.more
+
+import android.app.Activity
+import android.app.Instrumentation.ActivityResult
+import android.content.Intent
+import android.net.Uri
+import android.preference.PreferenceCategory
+import android.preference.PreferenceScreen
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.Intents.intending
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasAction
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasData
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasExtra
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasFlag
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import com.owncloud.android.BuildConfig
+import com.owncloud.android.R
+import com.owncloud.android.ui.activity.LogHistoryActivity
+import com.owncloud.android.ui.activity.Preferences
+import com.owncloud.android.ui.activity.PrivacyPolicyActivity
+import org.hamcrest.Matchers.allOf
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class OCSettingsMoreTest {
+
+    @Rule
+    @JvmField
+    val activityRule = ActivityTestRule(Preferences::class.java, true, true)
+
+    @Before
+    fun setUp() {
+        //Only interested in "More" section, so we can get rid of the other categories. SmoothScroll is not
+        //working fine to reach the bottom of the screen, so this approach was taken to display the section
+        val preferenceScreen = activityRule.activity.getPreferenceScreen() as PreferenceScreen
+        val cameraUploadsCategory = activityRule.activity.findPreference("camera_uploads_category") as PreferenceCategory
+        val securityCategory = activityRule.activity.findPreference("security_category") as PreferenceCategory
+        preferenceScreen.removePreference(cameraUploadsCategory)
+        preferenceScreen.removePreference(securityCategory)
+    }
+
+
+    @Test
+    fun helpView(){
+        onView(withText(R.string.prefs_help)).check(matches(isDisplayed()))
+    }
+
+    @Test
+    fun helpOptionOpensWeb(){
+        Intents.init()
+        onView(withText(R.string.prefs_help)).perform(click())
+        val intentResult = ActivityResult(Activity.RESULT_OK, Intent())
+        intending(hasAction(Intent.ACTION_VIEW)).respondWith(intentResult);
+        intended(hasData(activityRule.activity.getString(R.string.url_help)))
+        Intents.release()
+    }
+
+    @Test
+    fun dav5xView(){
```

Comment:
It would be davx5 but is not that important

Reply:
Corrected